core/frontend/src/tile/map/ImageryTileTree.ts

```diff
@@ -29,12 +29,23 @@ export interface ImageryTileContent extends TileContent {
 export class ImageryMapTile extends RealityTile {
   private _texture?: RenderTexture;
   private _mapTileUsageCount = 0;
+
+  /** True if the tile is outside the valid range of LOD : some tile tree might not be fully defined (i.e. TileTree might start at level 10+).

```

Comment:
This documentation better belongs on `Tile.isOutOfLodRange` than on this private field.